tests/integrations/celery/conftest.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+pytest.importorskip("celery")
```

Comment:
I believe moving this to `__init__` will fix your test issues. I assume it breaks a certain build that makes pytest skip all tests (and return with exitcode 1 because it didn't run anything)